assets/layouts/default.vue

```diff
@@ -112,11 +126,26 @@ function onResized(e: any) {
     left: -40px;
     width: 60px;
     padding-left: 40px;
+    color: white;
```

Comment:
Something doesn't sound right here. Colors shouldn't be hardcoded. Dozzle has light and dark mode. Can you make sure this is the right thing? Doing collapse in light mode looks like this

<img width="46" alt="Screen Shot 2022-11-08 at 9 37 53 AM" src="https://user-images.githubusercontent.com/260667/200636352-d7af735f-2d27-4494-9aa2-6588e25d9cd6.png">

That looks broken.

Reply:
Looks like in master I have bug. The background also turns black. Let's see if we can fix that in this PR. 

Is your intention to have the signout stick out? Why? Isn't the purpose of collapse to hide all buttons. User can uncollapse and click it if needed. No reason to take space.

Reply:
If there is no need to show the logout button when the panel is collapsed, then you can remove it and leave it only on the panel.

Reply:
I will try to fix the error with the button color.

Reply:
thank you ❤️